api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -81,4 +81,15 @@
 								 referencedTableName="concept"
 								 referencedColumnNames="concept_id" />
     </changeSet>
+    <changeSet id="TRUNK-5663-20191002" author="gitacliff">
```

Comment:
Because the master branch is on 2.4.0-SNAPSHOT, these changes would go in the liquibase-update-to-2.4.xml file

Reply:
@dkayiwa i have put the changes in liquibase-update-to-2.4.xml file